docs/security-and-privacy-design/netdata-cloud-security.md

```diff
@@ -1,116 +1,201 @@
 # Netdata Cloud Security and Privacy Design
 
-Netdata Cloud is designed with a security-first approach to ensure the highest level of protection for user data. When
-using Netdata Cloud in environments that require compliance with standards like PCI DSS, SOC 2, or HIPAA, users can be
-confident that all collected data is stored within their infrastructure. Data viewed on dashboards and alert
-notifications travel over Netdata Cloud, but aren’t stored—instead, they're transformed in transit, aggregated from
-multiple Agents and parents (centralization points), to appear as one data source in the user's browser.
+:::tip
+
+**Executive Summary**  
+Netdata Cloud offers secure real-time monitoring without storing raw metrics.  
+Only minimal metadata passes securely through Netdata Cloud.  
+Users retain full control of their data.  
+Infrastructure is protected wioth strong encryption, access control, and compliance with GDPR, CCPA, PCI DSS, SOC 2, and HIPAA standards.  
+Netdata Cloud continuously evolves its security and privacy practices to meet the highest industry standards.
```

Comment:
```suggestion
- Netdata Cloud offers secure real-time monitoring without storing raw metrics.  
- Only minimal metadata passes securely through Netdata Cloud.  
Users retain full control of their data.  
- Infrastructure is protected with strong encryption, access control, and compliance with GDPR, CCPA, PCI DSS, SOC 2, and HIPAA standards.  
- Netdata Cloud continuously evolves its security and privacy practices to meet the highest industry standards.
```

<img width="936" alt="Screenshot 2025-04-30 at 13 34 10" src="https://github.com/user-attachments/assets/a82e4365-846e-4358-a27d-329d1ce24827" />


=>

<img width="953" alt="Screenshot 2025-04-30 at 13 32 00" src="https://github.com/user-attachments/assets/11eaaf4a-db3b-47d1-92fe-409b574fe8b8" />

Reply:
The same applies to other summary tips.